### DISCUSSION TITLE
Toolbox when using autoloader?

### DISCUSSION THREAD

**Discussion by easyname889** [markmap/markmap#293] [Q&A] [answered]
Im running the below to autoload but i cant see the toolbox, I'm obviously missing something. How do i load the toolbar when using the autoloader?

`
        $markdown_content = file_get_contents( $sanitized_path );

        ?>
        <style>
            .markmap { position: relative; }
            .markmap > svg { width: 100%; height: 80vh; }
            svg.markmap {}
        </style>

        <div class="markmap">
            <script type="text/template"><?php echo esc_html( $markdown_content ); ?></script>
        </div>
        <script src="https://cdn.jsdelivr.net/npm/markmap-autoloader@latest"></script>
        <script src="https://cdn.jsdelivr.net/npm/markmap-toolbar@0.16.0"></script>

        <?php
        `

**Comment by gera2ld** [comment, marked as answer]
Add options before loading `markmap-autoloader` and no need to load `markmap-toolbar` explicitly:

```html
<script>
    window.markmap = {
        autoLoader: { toolbar: true },
    };
</script>
<script src="https://cdn.jsdelivr.net/npm/markmap-autoloader"></script>
```

Ref: https://markmap.js.org/docs/packages--markmap-autoloader#options

ps. I will update the options to include `toolbar` soon. You can find all available options here:
https://github.com/markmap/markmap/blob/abb51f061549c48a8770039360fc8cd5fa842719/packages/markmap-autoloader/src/types/index.d.ts#L4-L12
